### DISCUSSION TITLE
Export Vertices/Edges CSV from .nii file

### DISCUSSION THREAD

**Discussion by JordanYu21** [JacobBumgarner/VesselVio#65] [Q&A] [answered]
Is it possible to export a list of the vertices and edges (w/ additional data: radius, length, etc.) from a loaded .nii file ?

**Comment by JacobBumgarner** [comment]
Just to clarify, you're saying you want to load the `.nii` file, analyze it, and export a list of csv results?

**Reply by JordanYu21** [reply]
I am currently filtering and thresholding an image sequence in imageJ that allows me to export a .nii file of the altered image sequence. I'm able to visualize the volumetric view of the .nii file in VesselVio, but would like to export a list of edges and vertices from this .nii file.

**Reply by JacobBumgarner** [reply, marked as answer]
You can export a `graphml` object of vertices and edges. You could then independently convert this to a `csv` file. Hopefully that makes sense.
